runtime/centrifuge/src/lib.rs

```diff
@@ -494,6 +548,14 @@ pub enum ProxyType {
 	/// Deprecated ProxyType, that we are keeping due to the migration
 	_Staking,
 	NonProxy,
+	Borrow,
+	Price,
```

Comment:
I think this can be removed as there is no `price` step for loans anymore.

Reply:
Will remove Price then

Reply:
After loans is merged.